frame/curve-amm/src/tests.rs

```diff
@@ -3,12 +3,45 @@ use frame_support::assert_ok;
 use crate::mock::*;
 use composable_traits::dex::CurveAmm as CurveAmmTrait;
 use frame_support::traits::fungibles::{Inspect, Mutate};
+use proptest::prelude::*;
 use sp_runtime::{
+	helpers_128bit::multiply_by_rational,
 	traits::{Saturating, Zero},
 	FixedPointNumber, FixedU128, Permill,
 };
 use sp_std::cmp::Ordering;
 
+/// Accepts -2, -1, 0, 1, 2
+macro_rules! prop_assert_zero_epsilon {
```

Comment:
afaik @hussein-aitlahcen moved that to reusable codes

Reply:
Not yet in main sadly